charts/forms-flow-admin/templates/deployment.yaml

```diff
@@ -56,10 +101,9 @@ spec:
         - name: DATABASE_USERNAME
           valueFrom:
             secretKeyRef:
-            key: {{ .Values.ExternalDatabase.ExistingDatabaseUserNameKey }}
-            name: "{{ .Values.ExternalDatabase.ExistingSecretName }}"
-        {{- end }}
-        {{- if not .Values.ExternalDatabase.ExistingDatabaseUserNameKey }}
+              key: {{ .Values.ExternalDatabase.ExistingDatabaseUserNameKey }}
```

Comment:
Please format and optimise similar lines like below. All if else conditions which can be optimised like this can be done across all files @amal-k-soman-aot  : 

- name: DATABASE_NAME
  valueFrom:
    secretKeyRef:
      key: {{ .Values.ExternalDatabase.ExistingDatabaseNameKey | default "DATABASE_NAME" }}
      name: {{ .Values.ExternalDatabase.ExistingSecretName | default .Chart.Name }}